t/t5324-split-commit-graph.sh

```diff
@@ -210,8 +210,10 @@ test_expect_success 'test merge stragety constants' '
 		git config core.commitGraph true &&
 		test_line_count = 2 $graphdir/commit-graph-chain &&
 		test_commit 15 &&
-		git commit-graph write --reachable --split --size-multiple=10 --expire-time=1980-01-01 &&
+		touch -m -t 201801010000.00 $graphdir/extra.graph &&
```

Comment:
Should we also have a file the is newer than the expire time and make sure it is still there?

Reply:
That would be a good way to future-proof it (with an explicit file on each side)